### PR TITLE
Add serum, error test

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,4 @@ erl_crash.dump
 *.swo
 *~
 
-serum
+/serum
```
Comment (marocchino) [inline .gitignore]: Unlike the intention, it matched so broadly. So I changed to match only the top.

**Comment by Dalgona** [comment]
@marocchino Thanks! I'm working on some major code changes on a separate branch, so conflicts may occur if that branch is merged. I'll notify you later if that happens.

**Comment by marocchino** [comment]
No problem. 🙆‍♂️

**Comment by Dalgona** [comment]
Smoothly merged! Thanks! :tada::rocket:

**Pull request merged**